src/app/(proper_react)/(redesign)/(public)/LandingView.test.tsx

```diff
@@ -67,9 +67,11 @@ describe("When Premium is not available", () => {
     await user.click(submitButton[0]);
 
     expect(signIn).toHaveBeenCalledTimes(1);
-    expect(signIn).toHaveBeenCalledWith("fxa", expect.any(Object), {
-      email: "mail@example.com",
-    });
+    expect(signIn).toHaveBeenCalledWith(
+      "fxa",
+      expect.any(Object),
+      expect.stringContaining(`email=`),
```

Comment:
Removing the actual email address that was entered kinda neuters the test... I'm assuming this wouldn't break?

```suggestion
      expect.stringContaining(`email=mail@example.com`),
```

(And probably a test for the correct attribution parameters wouldn't harm too :) )

Reply:
Change the string match to `email=mail%40example.com` since it's encoded